protocols/gossipsub/src/peer_score.rs

```diff
@@ -274,11 +274,11 @@ impl PeerScore {
                         metrics.register_score_penalty(Penalty::MessageDeficit);
                     }
                     tracing::debug!(
-                        "[Penalty] The peer {} has a mesh message deliveries deficit of {} in topic\
-                         {} and will get penalized by {}",
-                        peer_id,
-                        deficit,
-                        topic,
+                        peer=%peer_id,
+                        %topic,
+                        %deficit,
+                        "[Penalty] The peer has a mesh message deliveries deficit in topic\
+                        and will get penalized by {}",
                         p3 * topic_params.mesh_message_deliveries_weight
```

Comment:
Let's make this value a structured log too! The calculation is in fact duplicated so we can extract it in a variable, see line 272.

Reply:
Let's call it the computed variable `penalty`.